### DISCUSSION TITLE
UNABLE TO PAY

### DISCUSSION THREAD

**Discussion by yardtechnology** [vercel/community#7303] [Help] [answered]
### Summary

An unexpected internal error occurred
<img width="1440" alt="vercel payment issue" src="https://github.com/vercel/community/assets/78586833/a6b54a95-645f-42bc-b2f5-caf414b39f51">

### Steps to Reproduce

An unexpected internal error occurred

**Comment by pawlean** [comment, marked as answer]
Hi @yardtechnology,

Thanks for reporting! Can you raise us a case [here](https://vercel.com/help)? Scroll down the page, click "Get Help" and follow the instructions. We'll get to it as soon as we can!

[a comment by an automated account is not shown]
